go/apps/http_api_nostream/resource.py

```diff
@@ -247,6 +247,13 @@ def handle_PUT_in_reply_to(self, request, payload, in_reply_to):
     @inlineCallbacks
     def handle_PUT_send_to(self, request, payload):
         user_account = request.getUser()
+        optout_store = self.worker.vumi_api.get_user_api(user_account) \
+            .optout_store
+        optout = yield optout_store.get_opt_out(
+            payload.get('to_addr_type', 'msisdn'), payload.get('to_addr'))
+        if optout:
+            self.client_error_response(request, "Recipient has opted out")
```

Comment:
From the slack discussion, happy with us using an error code here, since there doesn't see to be a success code that makes sense.